pkgs/applications/editors/ed/default.nix

```diff
@@ -1,21 +1,17 @@
 { fetchurl, stdenv }:
 
 stdenv.mkDerivation rec {
-  name = "ed-1.13";
+  name = "ed-${version}";
+  version = "1.14.1";
 
   src = fetchurl {
     # gnu only provides *.lz tarball, which is unfriendly for stdenv bootstrapping
     #url = "mirror://gnu/ed/${name}.tar.gz";
     # When updating, please make sure the sources pulled match those upstream by
     # Unpacking both tarballs and running `find . -type f -exec sha256sum \{\} \; | sha256sum`
     # in the resulting directory
-    urls = let file_md5 = "fb8ffc8d8072e13dd5799131e889bfa5"; # for fedora mirror
-      in [
-        ("http://pkgs.fedoraproject.org/repo/extras/ed"
-          + "/${name}.tar.bz2/${file_md5}/${name}.tar.bz2")
-        "http://fossies.org/linux/privat/${name}.tar.bz2"
-      ];
-    sha256 = "1iym2fsamxr886l3sz8lqzgf00bip5cr0aly8jp04f89kf5mvl0j";
+    url = "http://fossies.org/linux/privat/${name}.tar.gz";
```

Comment:
md5 is easy to find out now, so we can commit it, and hopefully Fedora will mirror it soon.